arch/xtensa/core/offsets/offsets.c

```diff
@@ -61,6 +61,11 @@ GEN_OFFSET_SYM(_xtensa_irq_bsa_t, fpu14);
 GEN_OFFSET_SYM(_xtensa_irq_bsa_t, fpu15);
 #endif
 
+#if (XCHAL_CP_MASK & (1 << XCHAL_CP_ID_AUDIOENGINELX)) && \
+	defined(CONFIG_CPU_HAS_DSP) && defined(CONFIG_DSP_SHARING)
+GEN_OFFSET_SYM(_xtensa_irq_bsa_t, ae_regs);
```

Comment:
any difference between different DSP architecture? @singalsu

Reply:
There's difference but I wonder if the aligned address and size macros used here cover those. I don't know.

Reply:
The aligned and size macros are defined in the HAL and should cover those differences.